app/templates/merchant/view_listings.html

```diff
@@ -109,6 +140,17 @@
                 updateCartQuantity(listingID, newQuantity);
                 updateCartFrontend(listingID, newQuantity);
             });
+
+        })
```

Comment:
What does this close? And if this should be here, add a semi-colon after the closing paren